### PR TITLE
Add Cursor.app v0.1.3 from https://www.cursor.so/

### PR DESCRIPTION
**Pull request by adzcai**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/cursor.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask "cursor" do
+  version "0.1.3"
+  sha256 :no_check
+
+  url "https://dl.todesktop.com/230313mzl4w4u92/mac/installer/universal",
+      verified: "dl.todesktop.com/230313mzl4w4u92"
+  name "Cursor"
+  desc "Build Software. Fast. Write, edit, and chat about your code with a powerful AI"
```
Comment (SMillerDev) [inline Casks/cursor.rb:8]: Let's remove all the marketing fluff
```suggestion
  desc "Write, edit, and chat about your code with AI"
```

File: Casks/cursor.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask "cursor" do
+  version "0.1.3"
```
Comment (SMillerDev) [inline Casks/cursor.rb:2]: Could you add a `livecheck` block for this

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/cursor.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    regex(/"Cursor",\n\s*"version":\s"(\d+(?:\.\d+)+)"/i)
+    strategy :page_match
+  end
+
+  depends_on macos: ">= :catalina"
+
+  installer script: "Install Cursor.app/Contents/MacOS/ToDesktop Installer"
+
+  uninstall quit:   "com.todesktop.230313mzl4w4u92",
+            delete: "/Applications/Cursor.app/"
```
Comment (reitermarkus) [inline Casks/cursor.rb:22]:
```suggestion
            delete: "/Applications/Cursor.app"
```

File: Casks/cursor.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  url "https://dl.todesktop.com/230313mzl4w4u92/mac/installer/universal",
+      verified: "dl.todesktop.com/230313mzl4w4u92/mac/installer/"
+  name "Cursor"
+  desc "Write, edit, and chat about your code with AI"
+  homepage "https://www.cursor.so/"
+
+  livecheck do
+    url "https://raw.githubusercontent.com/getcursor/cursor/main/package.json"
+    regex(/"Cursor",\n\s*"version":\s"(\d+(?:\.\d+)+)"/i)
+    strategy :page_match
```
Comment (reitermarkus) [inline Casks/cursor.rb:14]:
```suggestion
    strategy :json do |json|
      json["version"]
    end
```

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**